udata_hydra/crawl.py

```diff
@@ -26,20 +26,22 @@
 log = setup_logging()
 
 
+def is_valid_status(status):
+    if not status:
+        return False
+    status = int(status)
+    return status >= 200 and status < 400
+
+
 async def compute_check_has_changed(check_data, last_check) -> bool:
-    # FIXME: this is flawed (eg ssl.SSLCertVerificationError won't have a status)
-    is_first_check = last_check["status"] is None
-    status_has_changed = (
-        "status" in check_data
-        and check_data["status"] != last_check["status"]
-    )
+    is_first_check = not last_check
```

Comment:
If we don't want to check for `last_check` in every condition, we could compute criterions only if last_check exists? Not sure it would make it cleaner though

Reply:
Not sure either, that would make an ugly if or mass definitions. I quite like the idea that each criterion is independent too.